openapi/paths/reports@journal.yaml

```
@@ -0,0 +1,86 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a journal report
+  operationId: GetJournalReport
+  x-sdk-operation-name: getJournal
+  description: Retrieve a journal report.
+  parameters:
+    - name: currency
+      in: query
+      description: Revenue currency.
+      required: true
+      schema:
+        $ref: ../components/schemas/CurrencyCode.yaml
+    - name: bookedFrom
+      in: query
+      description: Revenue booked from month. Omit to retrieve booked starting from the start.
+      schema:
+        type: string
+        pattern: '^\d{4}-\d{2}$'
+        example: 2022-01
+    - name: bookedTo
+      in: query
+      description: Revenue booked to month. Omit to retrieve booked to the end.
+      schema:
+        type: string
+        pattern: '^\d{4}-\d{2}$'
+        example: 2022-01
+    - name: recognizedAt
+      in: query
+      description: Revenue recognition month.
+      required: true
+      schema:
+        type: string
+        pattern: '^\d{4}-\d{2}$'
+        example: 2022-04
+    - name: aggregationField
+      in: query
+      description: Report aggregation field.
+      required: true
+      schema:
+        type: string
+        enum:
+          - product.accountingCode
+          - product.id
+          - plan.id
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionFilter.yaml
+  responses:
+    '200':
+      description: Report was retrieved successfully.
```

nitpick: where possible, avoid past tense

```suggestion
      description: Report successfully retrieved.
```